tasks/configure.yml

```diff
@@ -54,9 +55,8 @@
    - "{{ tor_ports }}"
 
 - name: Abort if Ed25519 master keys are on the relay
-  fail: msg="
-
-            Ed25519 MASTER KEY detected on the relay - it is NOT supposed to be there! Aborting."
+  fail:
+    msg: "Ed25519 MASTER KEY detected on the relay - it is NOT supposed to be there! Aborting."
```

Comment:
I added these newlines back then to have the message be better visible from the rest, do the newlines no longer work?

Reply:
yeah, sorry about that, just assumed it was an accidental newline, reverted the format